Essentials/src/com/earth2me/essentials/Essentials.java

```diff
@@ -750,23 +752,32 @@ public int broadcastMessage(final IUser sender, final String message) {
         return broadcastMessage(sender, null, message, false);
     }
 
+    @Override
+    public int broadcastMessage(final IUser sender, final String message, final IUser... exclude) {
+        return broadcastMessage(sender, null, message, false, exclude);
+    }
+
     @Override
     public int broadcastMessage(final String permission, final String message) {
         return broadcastMessage(null, permission, message, false);
     }
 
-    private int broadcastMessage(final IUser sender, final String permission, final String message, final boolean keywords) {
+    private int broadcastMessage(final IUser sender, final String permission, final String message, final boolean keywords, final IUser... exclude) {
         if (sender != null && sender.isHidden()) {
             return 0;
         }
 
         IText broadcast = new SimpleTextInput(message);
 
         final Collection<Player> players = getOnlinePlayers();
+        final Set<IUser> excluded = new HashSet<>(Arrays.asList(exclude));
```

Comment:
If you're just going to make it a set anyways, wouldn't it be simpler to make the function argument a `final Set<IUser> excludedUsers` instead of using varargs? I'll admit that varargs are cool, but it might not be the right choice for this function, as I don't think it leaves as much flexibility for future changes.